autogen.sh

```diff
@@ -1,24 +1,24 @@
-#!/bin/sh
+#!/bin/env sh
 
-srcdir=`dirname $0`
+srcdir=$(dirname "$0")
 test -z "$srcdir" && srcdir=.
-ORIGDIR=`pwd`
+ORIGDIR="$(pwd)"
 
 set -x
 
-cd $srcdir
+cd "$srcdir" || exit
```

Comment:
i'd argue that just exiting without any error message or exit code is worse than running into the aclocal error that would happen when this fails.

Reply:
@rofl0r  wrote:
> i'd argue that just exiting without any error message or exit code is worse than running into the aclocal error that would happen when this fails.

Agreed. I changed to have a message along with exit.